CDCgov/prime-reportstream
# 13895 - Removed alerts from Daily Data and Data Dashboard

This PR ...
Removed alerts from Daily Data and Data Dashboard

Test Steps:
1. Log into RS
2. Select an org that has data in daily data
3. Go to /daily-data and see that the alert is no longer present
4. Go to /data-dashboard and see that the alert is no longer present

## Changes

![Screenshot 2024-05-22 at 9 21 17 AM](https://github.com/CDCgov/prime-reportstream/assets/102491809/14459d3f-6375-4ae9-ab94-19d74750832c)


![Screenshot 2024-05-21 at 4 07 03 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/360810cd-b4ad-4539-b790-c67b176caf88)

## Checklist

### Testing


## Linked Issues
- Fixes #13895
